File: Udemy/C#/C#_.NET/Exercicios/ExercicioFixacaoComClasse1/ExercicioFixacaoComClasse1/Retangulo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Globalization;

namespace ExercicioFixacaoComClasse1 {
    internal class Retangulo {
        
        CultureInfo CI = CultureInfo.InvariantCulture;

        public double Largura;
        public double Altura;


        public double CalculandoArea() {
            return Largura * Altura;
        }

        public double CalculandoPerimetro() {
            return Largura * 2 + Altura * 2  ;
        }

        public double CalculandoDiagona() { 
            return Math.Sqrt(Math.Pow(Largura,2)+Math.Pow(Altura,2));
        }

       // public override string ToString() {
      //      return "AREA = " + CalculandoArea().ToString("F2", CI) 
      //          + "PERÍMETRO = " + CalculandoPerimetro().ToString("F2",CI);
      //  }

    }
}

File: Udemy/Python/Exercicios/Retangulo.py
import math

base: float
altura: float
area: float
perimetro: float
diagonal: float

base = float(input("Base do retangulo: "))
altura = float(input("Altura do retangulo: "))

area = base * altura
perimetro = 2 * (base + altura)
diagonal = math.sqrt(base ** 2 + altura ** 2)

print(f"AREA = {area:.4f}")
print(f"PERIMENTRO = {perimetro:.4f}")
print(f"DIAGONAL = {diagonal:.4f}")

